src/user/profile.ts

```diff
@@ -11,23 +15,54 @@ export interface NDKUserProfile {
     lud16?: string;
     about?: string;
     zapService?: string;
+    website?: string;
 }
 
 export function mergeEvent(event: NDKEvent, profile: NDKUserProfile): NDKUserProfile {
     const payload = JSON.parse(event.content);
```

Comment:
This can throw an error if it's invalid JSON

Reply:
That's been there since the beginning. I don't think we need to have a custom error for this, the standard JSON parsing error is probably ok. @pablof7z?

Reply:
i was just wondering if it needed to be handled here.